lib/pengine/unpack.c

```diff
@@ -2802,6 +2802,7 @@ unpack_migrate_to_failure(pe_resource_t *rsc, pe_node_t *node, xmlNode *xml_op,
                      target_stop_id, target_migrate_from_id);
         if (target_node && target_node->details->online) {
             native_add_running(rsc, target_node, data_set, FALSE);
+            pe__set_resource_flags(rsc, pe_rsc_active_by_migrate_to);
```

Comment:
Of course a more precise way could be recording "target_node" which is the exact node where the resource is impliedly active by the migrate_to failure. But that would likely need pe_resource_s to be expanded even more...